industrial_ci/src/tests/clang_format_check.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (c) 2018, Jonathan Hechtbauer
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function clang_install() {
+  local version="$1"
+  sudo apt-get update -qq
+  sudo apt-get install -y -qq clang-format"$version"
+}
+
+function define_clang_format() {
+  if [ -f "$TARGET_REPO_PATH/.clang-format" ]; then
+    echo "Found '.clang-format' file."
+    CLANG_FORMAT="file"
+  else    
+    CLANG_FORMAT="$CLANGCHECK" # If not valid Clang uses default
+  fi
+}
+
+function run_clang_format_check() {
+  local version="${1:--3.8}" # default = -3.8
+  local ERR=0
+
+  ici_require_run_in_docker # this script must be run in docker
+  
+  ici_time_start clang_install
+    clang_install "$version"
+  ici_time_end # clang_install
+
+  ici_time_start define_clang_format
```

Comment:
No need to time this. I would prefer to branch the format without any function call.

Reply:
I removed the functions in 97bb8dc and b9885f4